### ISSUE TITLE
Create a Back-end Service to store all the book details entered by the user

### ISSUE THREAD

**Issue by SauravMukherjee44** [SauravMukherjee44/Aec-Library-Website#9]

**Comment by gautammali** [comment]
should i use node js to store the data and also mongoose for server

**Comment by gautammali** [comment]
can I use node js to store the data and mongoose as a server????

**Comment by SauravMukherjee44** [comment]
Yeah, you can.

**Comment by Sin-Sumit** [comment]
Hey, @SauravMukherjee44 Which type of language do we need to use for the backend?

**Comment by SauravMukherjee44** [comment]
PHP / NodeJs

**Comment by bharati-21** [comment]
@Sin-Sumit, Hi will you be taking up this issue?

**Comment by gautammali** [comment]
Yes I am talking about this and currently I am using the my local server to
store the data/connect to the server
Letter you will set the link of server as per requirement

**Comment by SauravMukherjee44** [comment]
@bharati-21 @gautammali Who will be taking this issue?

**Comment by gautammali** [comment]
I will complete the task by tomorrow or Wednesday

**Comment by gautammali** [comment]
Update for the backend  60 % of work is almost done. there are some issues I need to fix to run the app smoothly. possible by tomorrow I will send the PR

**Comment by gautammali** [comment]
hello sir update about the backend I was almost done with it but there are many small bugs in the previous code and also many typos that should be fixed to run the backend properly so I am working on it I will  try to complete it as soon as possible

**Comment by SauravMukherjee44** [comment]
Okay no worries

**Comment by priyam212001** [comment]
hey @SauravMukherjee44 , I want to work in this issue. Can I use Springboot as a backened language and mysql as a database. I have experience in this technology. Assign me this issue I will try to complete it as sonn as possible.
